libknet/bindings/rust/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "rust-kronosnet"
+version = "0.1.0"
+authors = ["Christine Caulfield <ccaulfie@redhat.com>"]
+edition = "2018"
+readme = "README.md"
+license = "MIT OR Apache-2.0"
```

Comment:
can we keep the license consistent with the rest of knet? GPLv2+ or LGPLv2+ ?

Reply:
by convention most rust crates are MIT and/or Apache, but of course you can use whatever license you want that is compatible with your dependencies. in this case, I'd advocate for LGPLv2.1+ since that is what libknet is licensed under, and having these bindings licensed more permissively does not make much sense to me? IANAL and all that ;)